backend/battle/tests.py

```diff
@@ -1,3 +1,46 @@
-# from django.test import TestCase
+from django.test import TestCase, Client
+from model_bakery import baker
+from users.models import User
+from battle.models import Battle
 
-# Create your tests here.
+'''
```

Comment:
Any reason for this to be commented? :eyes:

Reply:
nop, it's wrong. I'm going to delete it.